Index: .idea/sonarlint/securityhotspotstore/index.pb
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\nf\n6src/main/java/com/valueit/device/domaine/DeviceVo.java\u0012,3\\a\\3a9d09a16eaa1dbe06aa3865f0ec7bd73d7f2491\nq\nAsrc/main/java/com/valueit/device/domaine/EntrepriseConverter.java\u0012,a\\7\\a713d6f4f5a28b0e2b584f115ec61b6060bce093\ny\nIsrc/main/java/com/valueit/device/service/exception/BusinessException.java\u0012,5\\1\\517f8e7462daa46fdacd9e4128fe763d1c314dbc\nn\n>src/main/java/com/valueit/device/service/model/Entreprise.java\u0012,9\\0\\90841087a46ae5a21b4e1837692f3d59a2e8bcbf\nj\n:src/main/java/com/valueit/device/service/model/Device.java\u0012,2\\2\\22cbe55b06d8e094d6724f18dc2175d5cbcfe628\nj\n:src/main/java/com/valueit/device/dao/DeviceRepositoty.java\u0012,0\\f\\0fbcdee45b78ececada3639e5fc8b876ec637a52\nr\nBsrc/main/java/com/valueit/device/service/EntrepriseServiceImp.java\u0012,a\\a\\aaac0cc03c985f943db5c72582d7496b012e5df4\nl\n<src/test/java/com/valueit/device/DeviceApplicationTests.java\u0012,d\\1\\d1e23e234f2324b3226f7eebc89a582c845eccd9
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/sonarlint/securityhotspotstore/index.pb b/.idea/sonarlint/securityhotspotstore/index.pb
--- a/.idea/sonarlint/securityhotspotstore/index.pb	
+++ b/.idea/sonarlint/securityhotspotstore/index.pb	
@@ -1,17 +1,5 @@
 
-f
-6src/main/java/com/valueit/device/domaine/DeviceVo.java,3\a\3a9d09a16eaa1dbe06aa3865f0ec7bd73d7f2491
 q
 Asrc/main/java/com/valueit/device/domaine/EntrepriseConverter.java,a\7\a713d6f4f5a28b0e2b584f115ec61b6060bce093
 y
-Isrc/main/java/com/valueit/device/service/exception/BusinessException.java,5\1\517f8e7462daa46fdacd9e4128fe763d1c314dbc
-n
->src/main/java/com/valueit/device/service/model/Entreprise.java,9\0\90841087a46ae5a21b4e1837692f3d59a2e8bcbf
-j
-:src/main/java/com/valueit/device/service/model/Device.java,2\2\22cbe55b06d8e094d6724f18dc2175d5cbcfe628
-j
-:src/main/java/com/valueit/device/dao/DeviceRepositoty.java,0\f\0fbcdee45b78ececada3639e5fc8b876ec637a52
-r
-Bsrc/main/java/com/valueit/device/service/EntrepriseServiceImp.java,a\a\aaac0cc03c985f943db5c72582d7496b012e5df4
-l
-<src/test/java/com/valueit/device/DeviceApplicationTests.java,d\1\d1e23e234f2324b3226f7eebc89a582c845eccd9
\ No newline at end of file
+Isrc/main/java/com/valueit/device/service/exception/BusinessException.java,5\1\517f8e7462daa46fdacd9e4128fe763d1c314dbc
\ No newline at end of file
Index: .idea/sonarlint/issuestore/index.pb
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\nf\n6src/main/java/com/valueit/device/domaine/DeviceVo.java\u0012,3\\a\\3a9d09a16eaa1dbe06aa3865f0ec7bd73d7f2491\nq\nAsrc/main/java/com/valueit/device/domaine/EntrepriseConverter.java\u0012,a\\7\\a713d6f4f5a28b0e2b584f115ec61b6060bce093\ny\nIsrc/main/java/com/valueit/device/service/exception/BusinessException.java\u0012,5\\1\\517f8e7462daa46fdacd9e4128fe763d1c314dbc\nn\n>src/main/java/com/valueit/device/service/model/Entreprise.java\u0012,9\\0\\90841087a46ae5a21b4e1837692f3d59a2e8bcbf\nj\n:src/main/java/com/valueit/device/service/model/Device.java\u0012,2\\2\\22cbe55b06d8e094d6724f18dc2175d5cbcfe628\nj\n:src/main/java/com/valueit/device/dao/DeviceRepositoty.java\u0012,0\\f\\0fbcdee45b78ececada3639e5fc8b876ec637a52\nr\nBsrc/main/java/com/valueit/device/service/EntrepriseServiceImp.java\u0012,a\\a\\aaac0cc03c985f943db5c72582d7496b012e5df4\nl\n<src/test/java/com/valueit/device/DeviceApplicationTests.java\u0012,d\\1\\d1e23e234f2324b3226f7eebc89a582c845eccd9
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/sonarlint/issuestore/index.pb b/.idea/sonarlint/issuestore/index.pb
--- a/.idea/sonarlint/issuestore/index.pb	
+++ b/.idea/sonarlint/issuestore/index.pb	
@@ -1,17 +1,0 @@
-
-f
-6src/main/java/com/valueit/device/domaine/DeviceVo.java,3\a\3a9d09a16eaa1dbe06aa3865f0ec7bd73d7f2491
-q
-Asrc/main/java/com/valueit/device/domaine/EntrepriseConverter.java,a\7\a713d6f4f5a28b0e2b584f115ec61b6060bce093
-y
-Isrc/main/java/com/valueit/device/service/exception/BusinessException.java,5\1\517f8e7462daa46fdacd9e4128fe763d1c314dbc
-n
->src/main/java/com/valueit/device/service/model/Entreprise.java,9\0\90841087a46ae5a21b4e1837692f3d59a2e8bcbf
-j
-:src/main/java/com/valueit/device/service/model/Device.java,2\2\22cbe55b06d8e094d6724f18dc2175d5cbcfe628
-j
-:src/main/java/com/valueit/device/dao/DeviceRepositoty.java,0\f\0fbcdee45b78ececada3639e5fc8b876ec637a52
-r
-Bsrc/main/java/com/valueit/device/service/EntrepriseServiceImp.java,a\a\aaac0cc03c985f943db5c72582d7496b012e5df4
-l
-<src/test/java/com/valueit/device/DeviceApplicationTests.java,d\1\d1e23e234f2324b3226f7eebc89a582c845eccd9
\ No newline at end of file
Index: .idea/sonarlint/securityhotspotstore/index.pb
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\nf\n6src/main/java/com/valueit/device/domaine/DeviceVo.java\u0012,3\\a\\3a9d09a16eaa1dbe06aa3865f0ec7bd73d7f2491\nq\nAsrc/main/java/com/valueit/device/domaine/EntrepriseConverter.java\u0012,a\\7\\a713d6f4f5a28b0e2b584f115ec61b6060bce093\ny\nIsrc/main/java/com/valueit/device/service/exception/BusinessException.java\u0012,5\\1\\517f8e7462daa46fdacd9e4128fe763d1c314dbc\nn\n>src/main/java/com/valueit/device/service/model/Entreprise.java\u0012,9\\0\\90841087a46ae5a21b4e1837692f3d59a2e8bcbf\nj\n:src/main/java/com/valueit/device/service/model/Device.java\u0012,2\\2\\22cbe55b06d8e094d6724f18dc2175d5cbcfe628\nj\n:src/main/java/com/valueit/device/dao/DeviceRepositoty.java\u0012,0\\f\\0fbcdee45b78ececada3639e5fc8b876ec637a52\nr\nBsrc/main/java/com/valueit/device/service/EntrepriseServiceImp.java\u0012,a\\a\\aaac0cc03c985f943db5c72582d7496b012e5df4\nl\n<src/test/java/com/valueit/device/DeviceApplicationTests.java\u0012,d\\1\\d1e23e234f2324b3226f7eebc89a582c845eccd9
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/sonarlint/securityhotspotstore/index.pb b/.idea/sonarlint/securityhotspotstore/index.pb
--- a/.idea/sonarlint/securityhotspotstore/index.pb	
+++ b/.idea/sonarlint/securityhotspotstore/index.pb	
@@ -1,17 +1,5 @@
 
-f
-6src/main/java/com/valueit/device/domaine/DeviceVo.java,3\a\3a9d09a16eaa1dbe06aa3865f0ec7bd73d7f2491
 q
 Asrc/main/java/com/valueit/device/domaine/EntrepriseConverter.java,a\7\a713d6f4f5a28b0e2b584f115ec61b6060bce093
 y
-Isrc/main/java/com/valueit/device/service/exception/BusinessException.java,5\1\517f8e7462daa46fdacd9e4128fe763d1c314dbc
-n
->src/main/java/com/valueit/device/service/model/Entreprise.java,9\0\90841087a46ae5a21b4e1837692f3d59a2e8bcbf
-j
-:src/main/java/com/valueit/device/service/model/Device.java,2\2\22cbe55b06d8e094d6724f18dc2175d5cbcfe628
-j
-:src/main/java/com/valueit/device/dao/DeviceRepositoty.java,0\f\0fbcdee45b78ececada3639e5fc8b876ec637a52
-r
-Bsrc/main/java/com/valueit/device/service/EntrepriseServiceImp.java,a\a\aaac0cc03c985f943db5c72582d7496b012e5df4
-l
-<src/test/java/com/valueit/device/DeviceApplicationTests.java,d\1\d1e23e234f2324b3226f7eebc89a582c845eccd9
\ No newline at end of file
+Isrc/main/java/com/valueit/device/service/exception/BusinessException.java,5\1\517f8e7462daa46fdacd9e4128fe763d1c314dbc
\ No newline at end of file
Index: .idea/sonarlint/issuestore/index.pb
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\nf\n6src/main/java/com/valueit/device/domaine/DeviceVo.java\u0012,3\\a\\3a9d09a16eaa1dbe06aa3865f0ec7bd73d7f2491\nq\nAsrc/main/java/com/valueit/device/domaine/EntrepriseConverter.java\u0012,a\\7\\a713d6f4f5a28b0e2b584f115ec61b6060bce093\ny\nIsrc/main/java/com/valueit/device/service/exception/BusinessException.java\u0012,5\\1\\517f8e7462daa46fdacd9e4128fe763d1c314dbc\nn\n>src/main/java/com/valueit/device/service/model/Entreprise.java\u0012,9\\0\\90841087a46ae5a21b4e1837692f3d59a2e8bcbf\nj\n:src/main/java/com/valueit/device/service/model/Device.java\u0012,2\\2\\22cbe55b06d8e094d6724f18dc2175d5cbcfe628\nj\n:src/main/java/com/valueit/device/dao/DeviceRepositoty.java\u0012,0\\f\\0fbcdee45b78ececada3639e5fc8b876ec637a52\nr\nBsrc/main/java/com/valueit/device/service/EntrepriseServiceImp.java\u0012,a\\a\\aaac0cc03c985f943db5c72582d7496b012e5df4\nl\n<src/test/java/com/valueit/device/DeviceApplicationTests.java\u0012,d\\1\\d1e23e234f2324b3226f7eebc89a582c845eccd9
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/sonarlint/issuestore/index.pb b/.idea/sonarlint/issuestore/index.pb
--- a/.idea/sonarlint/issuestore/index.pb	
+++ b/.idea/sonarlint/issuestore/index.pb	
@@ -1,17 +1,0 @@
-
-f
-6src/main/java/com/valueit/device/domaine/DeviceVo.java,3\a\3a9d09a16eaa1dbe06aa3865f0ec7bd73d7f2491
-q
-Asrc/main/java/com/valueit/device/domaine/EntrepriseConverter.java,a\7\a713d6f4f5a28b0e2b584f115ec61b6060bce093
-y
-Isrc/main/java/com/valueit/device/service/exception/BusinessException.java,5\1\517f8e7462daa46fdacd9e4128fe763d1c314dbc
-n
->src/main/java/com/valueit/device/service/model/Entreprise.java,9\0\90841087a46ae5a21b4e1837692f3d59a2e8bcbf
-j
-:src/main/java/com/valueit/device/service/model/Device.java,2\2\22cbe55b06d8e094d6724f18dc2175d5cbcfe628
-j
-:src/main/java/com/valueit/device/dao/DeviceRepositoty.java,0\f\0fbcdee45b78ececada3639e5fc8b876ec637a52
-r
-Bsrc/main/java/com/valueit/device/service/EntrepriseServiceImp.java,a\a\aaac0cc03c985f943db5c72582d7496b012e5df4
-l
-<src/test/java/com/valueit/device/DeviceApplicationTests.java,d\1\d1e23e234f2324b3226f7eebc89a582c845eccd9
\ No newline at end of file
